tutorials/vue/vue-inject.md

```diff
@@ -0,0 +1,52 @@
+Vue's provide/inject API is a neat way to send props directly
+to the component you want while avoiding prop drilling.
+What you do is you declare what variables you will send, i.e. provide,
+in the setup property and then in the component that needs them you use the inject property.
```

Comment:
I'd create an example that doesn't use the `setup()` function first. You can just add an `inject` property to your components like we do in devtools. Then add a separate section that demonstrates how to use `inject()` with `setup()`.